src/app/components/header.tsx

```diff
@@ -61,6 +61,10 @@ export const Header: React.FC<HeaderProps> = memo(props => {
       justifyContent="space-between"
       p="base"
       minHeight={['', '80px']}
```

Comment:
Making this change as it looked like we needed a top margin but on review of the design we haven't been changing the colour of the header in extension view:  

![Screenshot 2023-09-11 at 08 43 14](https://github.com/leather-wallet/extension/assets/2938440/83106eda-d2cb-4757-894a-8f19d1b3a7e6)



OR this could be something that had been set by our base CSS